clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/templates/AzureServerGroupResourceTemplate.groovy

```diff
@@ -33,6 +33,7 @@ package com.netflix.spinnaker.clouddriver.azure.templates
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.SerializationFeature
 import com.netflix.spinnaker.clouddriver.azure.resources.servergroup.model.AzureServerGroupDescription.AzureInboundPortConfig
+import com.netflix.spinnaker.clouddriver.model.Image
 import groovy.util.logging.Slf4j
```

Comment:
Looks like the sort-order is off here.

Reply:
I found that this import is actually not used in the code. Removed it